resallocwebui/cgi-resalloc

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/python3
 from wsgiref.handlers import CGIHandler
-from resallocwebui.app import app
-CGIHandler().run(app)
+from resallocwebui.app import app as application
+
+
+# Apache only imports `application` from this file and does the rest
```

Comment:
The missing period makes this sentence hard to read, but that's just a nit.  +1, thank you.